app/models/accounts.py

```diff
@@ -12,32 +13,27 @@ class User(Base, UserMixin):
     Note: should only contain methods and fields
           related to this functionalties only
     '''
-    email = TextField()
+    email = TextField(primary_key=True)
     password = TextField()
 
     @staticmethod
     def register(email, password):
-        password = sha256_crypt.encrypt(password)
+        password = security.generate_password_hash(password, method='pbkdf2:sha1', salt_length=8)
         try:
             user = User.get(email=email)
-            if user:
-                # should raise an exception instead
-                raise "this user aleardy exists"
-        except: # an apropriate exception should be provided here
+            raise UserAlreadyExists
+        except User.DoesNotExist:
             user = User.create(email=email, password=password)
             return user
 
     @staticmethod
```

Comment:
why is it a static method
it should authenticate against the current user instance, it shouldn't get the user using his email
getting the user instance should be handled in the view instead
this implements a table wide functionality in an object instance